tests/common/include/tests/common/masks_cylinder3.inl

```diff
@@ -0,0 +1,53 @@
+/** Detray library, part of the ACTS project (R&D line)
```

Comment:
Wait, shoulkd this file not be deleted? ```cylinder3``` should not even exist as a mask

Reply:
Deleted